Windows/GEDebugger/CtrlDisplayListView.cpp

```diff
@@ -306,6 +310,10 @@ void CtrlDisplayListView::onMouseDown(WPARAM wParam, LPARAM lParam, int button)
 
 void CtrlDisplayListView::onMouseUp(WPARAM wParam, LPARAM lParam, int button)
 {
+	if (!validDisplayList) {
```

Comment:
Do we need both this and my state checks a few lines down there?

Reply:
Apparently we do, currently, but I'm changing it to check gpuDebug too. Now not needed.